enhancements/installer/openstack-no-fip-install.md

```diff
@@ -0,0 +1,78 @@
+---
+title: openstack-bring-your-own-external-connectivity
+authors:
+  - "@egarcia"
+  - "@adduarte"
+reviewers:
+  - "@mandre"
+  - "@pierre"
+approvers:
+  - "@mandre"
+  - "@pierre"
+creation-date: 2020-5-26
+last-updated: 2020-5-26
+status: implementable
+---
+
+# OpenStack BYO External Connectivity
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Numerous customers have requested the ability to run the OpenShift on OpenStack IPI installer without using Floating IP addresses.
+In the installer, we use floating IPs as a way to set up exeternal connectivity to the OpenShift cluster. Depending on how your cluster is 
+configured, the usage of floating IPs may not be possible or convenient in OpenStack. Our goal is to support configuring external 
```

Comment:
agree to remove the `external`, whether the new cluster is accessable to `external` is based on admin's setup
e.g have a `192.168` ip you are able to create a cluster if the installer on same subnet but it's not external accessable